tests/integration/test_remote_transfer.py

```
@@ -380,32 +418,38 @@ async def test_resumable_transfer(
     async def _send_chunks_partial(job, conn, db, client):
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `_send_chunks_partial`

Add return type annotation: `None`

(ANN202)